packages/sdk/src/testdata/txresponse1.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Nice samples

Maybe you can add to the readme how they were generated, so more could be added if needed (I think I know how, just nice to doc)